README.md

```diff
@@ -51,17 +51,17 @@ In witness automata, the GraphML edges represent state transitions.
+| assumption.scope | *Valid values:* Function name <br /> ``assumption.scope`` is used to qualify variables with ambiguous names in a state-space guard by specifying a function name: The consumer first has to look for a variable with a matching name in the scope of the function with the name specified via a ``data`` element with the ``assumption.scope`` key before checking the global scope. This key always applies to the state-space guard as a whole, i.e., it is not possible to specify a state-space guard over local variables of different functions. In existing implementations, there is currently no support for different variables with the same name within different scopes of the same function. This key is not allowed in correctness witnesses. The ``attr.type`` attribute of this key is ``string.`` | Yes | No |
+| assumption.resultfunction | *Valid values:* Function name <br /> ``assumption.resultfunction`` is used to specify the function of the ``\result`` variable used in a state-space guard of the same transition, meaning that ``\result`` represents the return value of the given function. This key applies to the state-space guard as a whole, it is therefore not possible to refer to multiple function-return values within the same transition. If the ``\result`` variable is used, a ``data`` element with this key must be used in the same transition, otherwise it is superfluous. This key is not allowed in correctness witnesses. The ``attr.type`` attribute of this key is ``string``. | Yes | No |
+| control | *Valid values:* ``condition-true`` or ``condition-false`` <br /> ``control`` is used as part of the source-code guard of a transition and restricts the set of CFA edges matched by the source-code guard to assume operations of the CFA. Valid values for ``data`` elements with this key are ``condition-true`` and ``condition-false``, where ``condition-true`` specifies the branch where the assume condition evaluates to ``true``, i.e., the ``then`` branch, and ``condition-false`` specifies the branch where the assume condition evaluates to ``false``, i.e., the ``else`` branch. The ``attr.type`` attribute of this key is ``string``. | Yes | Yes |
+| startline | *Valid values:* Valid line number of the program <br /> ``startline`` is used as part of the source-code guard of a transition and restricts the set of CFA edges matched by the source-code guard to operations on specific lines in the source code. Any line number of the source code is a valid value for ``data`` elements with this key. A ``startline`` refers to the line number on which an operation of a CFA edge begins. The ``attr.type`` attribute of this key is ``int``. | Yes | Yes |
```

Comment:
This is potentially a breaking change, since the ``attr.type`` was not fixed to ``int`` before. We should be able to check this against the SV-COMP's witnesses to see whether this poses any problems. ``attr.type`` is also fixed for other keys, the same applies there.

Reply:
I do not anticipate any practical issues with this change. That said, breaking changes are sometimes necessary. This is how the upcoming paper will define the startline key.